ccube.nim

```diff
@@ -14,23 +14,23 @@ Distributed under GPLV3
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ]#
-import std/[os,strutils,strformat,random,json]
+import std/[os, strutils, strformat, random, json, parseopt]
 import bigints
 
 randomize()
 
-let VER = "1.0.0"
-let banner = fmt"""
+const VER = "1.0.0"
+const banner = fmt"""
```

Comment:
Since you don't change those values at all, it's better to have them as `const` - https://nim-lang.org/docs/manual.html#constants-and-constant-expressions